CONTRIBUTING.md

```diff
@@ -42,6 +42,11 @@ Add one `Describe` section per method or function that you are unit testing:
 Within a `Describe`, use one `It` section per test case.
 If several `It` sections are similar, consider refactoring them in a `DescribeTable` construct with varying inputs.
 
+### Generating bundles for OperatorHub.io
+
+Use `hack/release-operator-hub` to generate bundles to be imported in
```

Comment:
```suggestion
Use `hack/release-operatorhub` to generate bundles to be imported in
```

`operator-hub` can have different meanings and is not very clear.

Reply:
How about running the script directly in the Makefile in the `bundle` target instead of documenting it?

Reply:
We could make a Makefile target, but what would be the benefit? You still to provide all the variables needed.